Assets/Scripts/Tank/TankModel.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace TankBattle.Tank
+{
+    public class TankModel
+    {
+        private float speed;
+        public float Speed {get {return speed; }}
+        private float turningTorque;
+        public float TurningTorque {get{ return turningTorque; }}
+        private int health;
+        public int Health {get{ return health;} set{health = value; }}
+        private TankType tankType;
+        private TankView tankPrefab;
+        public TankView TankPrefab{get{return tankPrefab; }}
+        private int kills;
+        public int Kills{get{return kills; } set{kills = value; }}
```

Comment:
if you have a public getter and setter then you don't need the private variables for such properties. you can just do - 
public int Kills { get; set; }